requirements.txt

```diff
@@ -3,3 +3,4 @@
 # SPDX-License-Identifier: Unlicense
 
 Adafruit-Blinka
+Adafruit-Circuitpython-ConnectionManager@git+https://github.com/justmobilize/Adafruit_CircuitPython_ConnectionManager@connection-manager
```

Comment:
This sort of stuff makes me wary as it might signal that this library might not receive enough maintenance in the long term. Having something published as a official package would be much more trustworthy in my opinion.

Reply:
This is a work in progress PR, it will use `Adafruit-Circuitpython-ConnectionManager` when it is released.

Reply:
Since `ConnectionManager` is merged. This has been changed to pull from pypi